src/reducers/authReducer.js

```diff
@@ -0,0 +1,17 @@
+import { SIGN_IN, SIGN_OUT } from '../actions/types';
+
+const INITIAL_STATE = {
+  isSignedIn: null,
+  accessToken: null
+};
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case SIGN_IN:
+      return { ...state, isSignedIn: true, accessToken: action.payload };
+    case SIGN_OUT:
```

Comment:
If we are setting everything to INITIAL_STATE, can't we just return INITIAL_STATE